src/docs/product/performance/mobile-vitals.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Mobile Vitals"
+sidebar_order: 60
+description: "Learn more about Mobile Vitals in Sentry, and how they give you a better understanding of your mobile application's health."
+---
+
+Mobile Vitals are a set of metrics to measure how fast your app starts, the duration of HTTP requests, the number of slow and frozen frames, and more. Each metric provides insights into the overall performance of your mobile application. These metrics give you the details you need to not only prioritize critical performance issues, but to trace issues down to the root cause to solve them faster.
```

Comment:
Perhaps I misunderstood, but the mobile vitals do not measure the duration of HTTP requests. We only have them to measure app starts, slow/frozen frames, and there's JS stalls for react-native.

Reply:
correct, we also track the duration of HTTP requests, but that's not part of the Mobile vitals afaik